api/src/pcapi/core/bookings/repository.py

```diff
@@ -832,3 +835,22 @@ def _serialize_csv_report(query: Query) -> str:
         )
 
     return output.getvalue()
+
+
+def get_unretrieved_booking_ids() -> typing.Generator[int, None, None]:
+    three_days_ago = datetime.combine(date.today() - timedelta(days=3), time(0, 0))
+
+    bookings = (
+        Booking.query.join(Stock, Offer)
+        .filter_by(canExpire=True)
+        .filter(Booking.isUsed.is_(False))
+        .filter(Booking.isCancelled.is_(False))
+        .filter(Booking.dateCreated >= three_days_ago)
```

Comment:
Ce filtre récupère les réservations qui ont été créés après "il y a trois jours" (donc plus jeunes que 3 jours). On veut l'inverse, non : les réservations créées **avant** (c'est-à-dire plus vieilles que 3 jours), non ?

```suggestion
        .filter(Booking.dateCreated < three_days_ago)
```